### ISSUE TITLE
No audio indication of incoming XMPP message from Ubuntu Linux user (unless you rename the contact?!)

### ISSUE THREAD

**Issue by tv21** [tigase/beagle-im#158]
> # NOTE: Tigase BeagleIM has new home at [tigase.dev/tigase/_clients/beagle-im/](https://tigase.dev/tigase/_clients/beagle-im/) - please submit all issues/pull-requests there!

**Describe the bug**
Beagle IM does not produce any kind of sound or audible alert when it receives a XMPP message from a Ubuntu Linux user, even though it shows the message and shows a visual indication on the dock icon.

**To Reproduce**

Have Beagle IM running and have someone send you a XMPP chat message from a Linux program such as Gajim, Dino, Kaidan, etc.

**Expected behavior**
It should make some kind of alert sound, in addition to showing the visual indication.

**Desktop (please complete the following information):**
 - OS: MacOS Ventura
 - Browser N/A (no 
 - Version 5.3.5 (178)

**Additional context**
This does not seem to be an issue if the other user is using a Mac, it only seems to happen if they are using Linux (specifically Ubuntu 22.04 in this case) and just by accident it was discovered that if you rename the contact then the alert sound starts to work.  I have no idea what renaming the contact has to do with the alert sound but that seems to have fixed the problem, at least in this one instance.

**Comment by hantu85** [comment]
I do not think this has anything to do with renaming contact. Actually, BeagleIM is not producing audio alert if the app has focus. I suppose that this is most likely the cause of the issue you are experiencing.
